community/kernel/src/main/java/org/neo4j/kernel/configuration/Config.java

```diff
@@ -610,10 +622,15 @@ public void updateDynamicSetting( String setting, String update )
                 }
 
                 oldValue = getDefaultValueOf( setting );
-                params.put( setting, update );
+                if ( params.put( setting, update ) == null )
+                {
+                    oldDefault = true;
+                }
                 newValue = update;
             }
-            log.info( "Setting changed: '%s' changed from '%s' to '%s'", setting, oldValue, newValue );
+            log.info( "Setting changed: '%s' changed from '%s' to '%s'",
```

Comment:
there is no new test for different output, also existent test that checks output now failing

Reply:
ah, this was backported from 3.4, missed the test...